Makefile

```diff
@@ -1,16 +1,21 @@
-DICTIONARIES := codespell_lib/data/dictionary*.txt codespell_lib/tests/data/*.wordlist
+DICTIONARIES = codespell_lib/data/dictionary*.txt codespell_lib/tests/data/*.wordlist
 
 PHONY := all check check-dictionaries sort-dictionaries trim-dictionaries check-dist pytest pypi ruff clean
 
-all: check-dictionaries codespell.1
+GENERATED := codespell_lib/data/dictionary_en_to_en-OX_AUTOGENERATED.txt
 
-check: check-dictionaries check-dist pytest ruff
+all: $(GENERATED) check-dictionaries codespell.1
+
+check: $(GENERATED) check-dictionaries check-dist pytest ruff
+
+$(GENERATED): codespell_lib/data/dictionary_en-GB_to_en-US.txt ./codespell_lib/gen_OX.sh
```

Comment:
I am not good at makefiles... is there a `make <something>` I could use to run this? If so please fix the `autofix.ci` which will just duplicate the command below...